extension/notification/src/main/resources/locale/notification/template/Notification_en.properties

```diff
@@ -278,3 +278,15 @@ UINotification.title.DlpUserRestoredItemPlugin=Item has been validated
 Notification.title.DlpUserRestoredItemPlugin=Item has been validated
 Notification.subject.DlpUserRestoredItemPlugin=Item has been validated
 Notification.intranet.message.DlpUserRestoredItemPlugin=The item named <strong>{0}</strong> has been analyzed during a quarantine phase. It has been validated and restored in his initial location.
+
+
+#############################################################################
+#                         MalwareDetectionPlugin                         #
+#############################################################################
+UINotification.label.group.MalwareDetection=Malware detection
+UINotification.label.MalwareDetectionPlugin=Malware detection
+UINotification.title.MalwareDetectionPlugin=Item is infected
```

Comment:
should we simply replace "item" by "file" or "document" ? seems more user friendly

Reply:
I have used the same label of DLP, I will see with PO to put the good label

Reply:
Modified @plamarque